source/extensions/filters/common/expr/evaluator.h

```diff
@@ -104,6 +131,17 @@ class CompiledExpression {
   static absl::StatusOr<CompiledExpression> Create(const BuilderInstanceSharedConstPtr& builder,
                                                    const cel::expr::Expr& expr);
 
+  // Creates an interpretable expression with custom configuration.
+  static absl::StatusOr<CompiledExpression>
+  Create(Server::Configuration::CommonFactoryContext& context, const cel::expr::Expr& expr,
+         const envoy::config::core::v3::CelExpressionConfig* config);
```

Comment:
I think you can just extend the previous stubs with OptRef and default set to {}.

Reply:
Make sense, Thanks!